charters/DataPortal/charter.md

```diff
@@ -0,0 +1,78 @@
+# Data Portal
+
+
+## Description
+The Data Portal is a landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides the user experience for scientific and general users to upload, explore, search, and run analyses on human genomic data. The Data Browser ("Explore") application provides a faceted search capability to find and download subsets of this data.
+
+## Objectives
+The objective of the Data Portal is to provide the DCP website implementation. It provides the access point for scientific and general users to search all HCA genetic data in order to identify a subset of the data for download and/or further analysis, and file/bundle search with public APIs at petabyte scale on public clouds.
+
+## In-scope
+### General
+* UX, visual, and branding design with input from the HCA brand working group
+* Adhere to the HCA DCP community best practices for Continuous Integration (CI), Continuous Delivery (CD), and system logging via Google Analytics, etc.
+* Production operation of the overall Data Portal website in Amazon Web Services (AWS), with exclusions listed in the out-of-scope section
+* Meet the performance scaling objectives of the DCP
+* DevSecOps compliance as required by the HCA Security Team
+* Leverage the authentication and authorization implementation libraries from the DevSecOps team to implement login and access control for the Data Portal and Data Browser
+* Community engagement:
+   * Triage and integration of feature requests into the work queue 
+   * Support plan for escalation of user reported issues
+### Data Portal
+* Select and deploy a content management system
+* Integrate Data Browser and Data Portal UIs into a seamless UX and provide links to other groups' applications (e.g. the Ingestion Service UI)
+* Frequent updates of summary statistics (cells, organs, donors, projects, labs, etc.) on the DCP home page
+* Coordinate development of user guide content (e.g. “Learn”) from multiple contributors
+* Coordinate development of developer documentation content (e.g. “Develop”) from multiple contributors. Includes API documentation and development guides.
+* Click through to Data Browser from search box and human body image
+* Registration form for tertiary portals and list of portals available with click through to hosting web site (e.g. Dockstore)
+* Integration with help desk, discussion forum, and user feedback applications
```

Comment:
The Data Portal will also offer click through to the data submission UI from the Contribute section (though it doesn't at the moment).

Reply:
Added to In-Scope section and to Milestones with a 1H 2019 delivery.